modules/oe_content_news/config/install/core.entity_form_display.node.oe_news.default.yml

```diff
@@ -7,17 +7,55 @@ dependencies:
     - field.field.node.oe_news.oe_news_featured_media
     - field.field.node.oe_news.oe_news_location
     - field.field.node.oe_news.oe_publication_date
+    - field.field.node.oe_news.oe_reference_code
     - field.field.node.oe_news.oe_related_links
     - field.field.node.oe_news.oe_subject
     - field.field.node.oe_news.oe_summary
     - field.field.node.oe_news.oe_teaser
     - node.type.oe_news
   module:
     - datetime
+    - field_group
     - link
     - maxlength
+    - path
```

Comment:
We should remove this dependency and related field from all default displays, I see it's also on the other content type. @yenyasinn  could you please create a follow up for this?

Reply:
Follow up has been created https://citnet.tech.ec.europa.eu/CITnet/jira/browse/EWPP-291